input/cql/PCSBPScreeningFollowUpFHIR.cql

```diff
@@ -309,8 +325,11 @@ define "Second Hypertensive Reading SBP 130 to 139 OR DBP 80 to 89 Interventions
 define "Second Hypertensive Reading SBP Greater than or Equal to 140 OR DBP Greater than or Equal to 90 Interventions Declined":
   ( ( ( [ServiceNotRequested: "Referral to Primary Care or Alternate Provider"]
         union [ServiceNotRequested: "Follow Up Within 4 Weeks"] ) SecondHTN140Over90ReferralFollowUpNotDone
-        where SecondHTN140Over90ReferralFollowUpNotDone.reasonRefused in "Patient Declined"
-          and SecondHTN140Over90ReferralFollowUpNotDone.status = 'completed'
+        where SecondHTN140Over90ReferralFollowUpNotDone.status = 'completed'
+          and exists(SecondHTN140Over90ReferralFollowUpNotDone.extension E 
+            where E.url = 'http://hl7.org/fhir/us/qicore/StructureDefinition/qicore-doNotPerformReason' 
+            and E.value as Code in "Patient Declined"
+          )
```

Comment:
I think this is expected to work as `SecondHTN140Over90ReferralFollowUpNotDone.reasonRefused`

Reply:
Correct